R/pkg/R/serialize.R

```diff
@@ -71,18 +69,19 @@ writeObject <- function(con, object, writeType = TRUE) {
   switch(serdeType,
          NULL = writeVoid(con),
          integer = writeInt(con, object),
+         glue =,
          character = writeString(con, object),
          logical = writeBoolean(con, object),
-         double = writeDouble(con, object),
+         double =,
```

Comment:
I think it's better to keep it anyway. Looks previous one is better to read.

Reply:
readability is in the eye of the beholder 🙃 

anyway this `switch` will go away after 28070

Reply:
k. but let's keep only related changes in case we backport or revert. that will likely reduce the conflicts when we do.